app/views/ResultsPageView.scala.html

```diff
@@ -46,7 +46,7 @@ <h2 class="govuk-heading-m" style="margin-bottom: 7px;">@messages("resultsPage.c
     }
 
     <div class="app-table">
-        @{replaceTableHeader(displayCorporationTaxTable(calculatorResult))}
+        @{replaceTableHeader(messages("resultsPage.corporationTaxTableSummary"),displayCorporationTaxTable(calculatorResult))}
```

Comment:
corporationTaxTableSummary - can we more clear that its screen reader summary i.e corporationTaxTableScreenReaderSummary ?

Reply:
Also, cant we move the replacement to the displayCorporationTaxTable method itself. Also applies to displayEffectiveTaxTable